index.html

```diff
@@ -40,4 +40,234 @@
+        <img class="work-img" src="Microverse-Student-Project-1/img2.png" alt="photo">
+      </div>
+      <div class="left-block">
+        <div class="primary-text">
+          <h1 class="project-title">Tonic</h1>
+          <div class="frame-2">
+            <div class="Client">
+              <h2 class="CANPOY">CANPOY</h2>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">2022</h2></div>
+            </div>
+          </div>
+        </div>
+        <div class="information">
+          <p class="des-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        
+        <ul class="tags">
+          <li class="tag"> <a class="html" href="">html</a></li>
+          <li class="tag"> <a class="html" href="">CSS</a></li>
+          <li class="tagj"> <a class="html" href="">JavaScript</a></li>
+        </ul>
+        
+        <div class="action">
+         <button class="normal" type="button">See Project</button>
+        </div>
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+  </section>
+
+  <section class="section-4">
+    <div class="card">
+      <div class="photo">
+        <img class="work-img" src="Microverse-Student-Project-1/img3.png" alt="photo">
+      </div>
+      <div class="left-block">
+        <div class="primary-text">
+          <h1 class="project-title">Multi-Post
+            Stories</h1>
+          <div class="frame-2">
+            <div class="Client">
+              <h2 class="CANPOY">CANPOY</h2>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">Back End Dev</h2></div>
+              <i class="counter fa-solid fa-circle"></i>
+              <div class="Role"><h2 class="back-end">2022</h2></div>
+            </div>
+          </div>
+        </div>
+        <div class="information">
+          <p class="des-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        
+        <ul class="tags">
+          <li class="tag"> <a class="html" href="">html</a></li>
+          <li class="tag"> <a class="html" href="">CSS</a></li>
+          <li class="tagj"> <a class="html" href="">JavaScript</a></li>
+        </ul>
+        
+        <div class="action">
+         <button class="normal" type="button">See Project</button>
+        </div>
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+  </section>
+
+
+  <section class="section-4">
+    <div class="card">
+      <div class="photo">
+        <img class="work-img" src="Microverse-Student-Project-1/img4.png" alt="photo">
+      </div>
+      <div class="left-block">
+        <div class="primary-text">
+          <h1 class="project-title">Tonic</h1>
```

Comment:
- [ ] As `Tonic` is a subtitle, it's a good practice to use the `hx` tag, but the `<h1>` tag is only used for the main title. Kindly use another `<hx>` instead of `<h1>` eg:(h2, h4, h5...).